scene/resources/bit_map.cpp

```diff
@@ -32,13 +32,18 @@
 
 #include "core/io/image_loader.h"
 
-void BitMap::create(const Size2 &p_size) {
+void BitMap::create(const Size2i &p_size) {
 	ERR_FAIL_COND(p_size.width < 1);
 	ERR_FAIL_COND(p_size.height < 1);
 
+	ERR_FAIL_COND(static_cast<int64_t>(p_size.width) * static_cast<int64_t>(p_size.height) > INT32_MAX);
+
+	Error err = bitmask.resize((((p_size.width * p_size.height) - 1) / 8) + 1);
+	ERR_FAIL_COND(err != Error::OK);
```

Comment:
We don't usually scope error codes with `Error::`, so this is pretty inconsistent with the rest of the codebase.

Reply:
Fixed.